dex/src/main/scala/com/wavesplatform/dex/AddressWsMutableState.scala

```diff
@@ -78,7 +79,10 @@ case class AddressWsMutableState(address: Address,
     }
   )
 
-  def cleanChanges(): AddressWsMutableState = copy(changedSpendableAssets = Set.empty, changedReservableAssets = Set.empty, ordersChanges = Map.empty)
+  def cleanAllChanges(): AddressWsMutableState =
```

Comment:
`cleanOrderChanges().cleanBalanceChanges()`?

Reply:
A'm just doing less copies in a memory (lol) :)